app/main.py

```diff
@@ -1,3 +1,52 @@
 class Distance:
-    # Write your code here
-    pass
+    def __init__(self, km: (int, float)):
```

Comment:
I'm not sure that it's correct to use (int, float) as type annotation in this case. I have always used Union[int, float].
Everything else is good. Well done!

Reply:
agree, use `int | float` or `Union[int, float]`